### DISCUSSION TITLE
Got error while building packages

### DISCUSSION THREAD

**Discussion by John4650-hub** [termux/termux-packages#14335] [Q&A] [answered]
configure: error: in `/root/.termux-build/nano/build':
configure: error: C compiler cannot create executables

I don't know what caused this error because i was able to build vim ,zip and some other packages but when i tried to build nodejs-lts, the error msg cameup, I have also tried to build nano but still the same error. Help.
Am using github actions due to small disk space

**Comment by sylirre** [comment, marked as answer]
You are doing something really wrong when compiling packages. Please build packages using our Docker image, if not familiar with environment configuration and software build process.

If using GitHub Actions, then refer to this configuration: https://github.com/termux/termux-packages/blob/master/.github/workflows/packages.yml
